cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -498,17 +498,17 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 		assert!(
 			events.iter().any(|event| matches!(
 				event,
-				RuntimeEvent::Balances(pallet_balances::Event::Minted { who, amount })
-					if *who == TREASURY_ACCOUNT.into() && *amount == 16903333
```

Comment:
dq; is there a way to generate programmatically the fee based on weights?

Reply:
You could call the fee calculation manually, but then you compare the same outputs of the same function.